### PR TITLE
[14.0][ADD] purchase_stock_vendor_bill_breakdown

### PR DESCRIPTION
**Pull request by geomer198**
This module allows to use module ‘purchase_vendor_bill_breakdown’  along with the module 'stock'.

### CODE REVIEW

**Review by francesco-ooops — APPROVED** [review]
functional ok!

this is the stock version of https://github.com/OCA/purchase-workflow/tree/14.0/purchase_vendor_bill_breakdown

File: purchase_stock_vendor_bill_breakdown/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    "name": "Purchase Stock Vendor Bill Product Breakdown",
+    "summary": "Purchase Stock Vendor Bill Product Breakdown",
+    "author": "Ooops, Cetmix, Odoo Community Association (OCA)",
+    "version": "14.0.1.0.0",
+    "category": "Purchase Management",
+    "website": "https://github.com/OCA/purchase-workflow",
+    "depends": ["purchase_vendor_bill_breakdown", "stock"],
+    "maintainers": ["geomer198", "CetmixGitDrone"],
+    "license": "AGPL-3",
+    "installable": True,
```
Comment (GabbasovDinar) [inline purchase_stock_vendor_bill_breakdown/__manifest__.py:11]: I understand that this module will be used for module compatibility (purchase_vendor_bill_breakdown and stock), maybe in this case would add the parameter "auto_install": True?

File: purchase_stock_vendor_bill_breakdown/models/purchase_order_line.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+class PurchaseOrderLine(models.Model):
+    _inherit = "purchase.order.line"
+
+    @api.depends("move_ids.state", "move_ids.product_uom_qty", "move_ids.product_uom")
+    def _compute_qty_received(self):
+        super(PurchaseOrderLine, self)._compute_qty_received()
+        for line in self.filtered(lambda l: l.qty_received and l._has_components()):
+            # update product components qty
+            line.component_ids._update_qty(line.qty_received - line.last_qty_invoiced)
```
Comment (GabbasovDinar) [inline purchase_stock_vendor_bill_breakdown/models/purchase_order_line.py]: Please, don't update the field of another model in a method that is not intended for this purpose, this method is only needed to compute the `qty_received` field. Maybe you could use the `def _write` method in your case?

**Review by GabbasovDinar — CHANGES_REQUESTED** [review]

File: purchase_stock_vendor_bill_breakdown/__manifest__.py (hunk 1 above)
Comment (geomer198) [reply purchase_stock_vendor_bill_breakdown/__manifest__.py:11]: Resolved

**Review by GabbasovDinar — APPROVED** [review]
Code LGTM! Thanks

**Comment by GabbasovDinar** [comment]
@geomer198 please, squash commits

**Comment by francesco-ooops** [comment]
@OCA/procurement-purchase-maintainers this is the stock version of https://github.com/OCA/purchase-workflow/tree/14.0/purchase_vendor_bill_breakdown

do you think it's good for merge? both functional and code review have been done

**Pull request merged**